Sources/RealDeviceMap/Controller/WebHookController.swift

```diff
@@ -238,7 +245,9 @@ class WebHookController {
             .addHeader(.contentType, "application/json"),
             .addHeader(.accept, "application/json"),
             .addHeader(.cacheControl, "no-cache"),
-            .addHeader(.userAgent, "RealDeviceMap \(VersionManager.global.version)")
+            .addHeader(.userAgent, "RealDeviceMap \(VersionManager.global.version)"),
```

Comment:
@Fabio1988 do you need the `\` here or can it be removed? (or is there one missing at the end?) 😃